### ISSUE TITLE
Mac: Calling Pause and deactivating the MediaPlayer on the same frame incorrectly causes playback to resume when you reactivate

### ISSUE THREAD

**Issue by TheloTheGreat** [RenderHeads/UnityPlugin-AVProVideo#674]
On Mac AVPro 2.0.9, but not on Windows, calling Pause and deactivating the MediaPlayer on the same frame incorrectly causes playback to resume when you reactivate.

Repro: In Demo_MediaPlayer on Mac, add this to MediaPlayerUI.Update:

```
if (Input.GetKeyDown(KeyCode.A)) {
	_mediaPlayer.Control.Pause();
	_mediaPlayer.gameObject.SetActive(false);
}
if (Input.GetKeyDown(KeyCode.S)) {
	_mediaPlayer.gameObject.SetActive(true);
}
```

Then while the MediaPlayer is playing, press A to pause and deactivate it. Then press S to reactivate it.

Expected result: The MediaPlayer stays paused.
Actual result: The MediaPlayer resumes playback.

Note: If you pause, then on a later frame you deactivate the gameobject, the issue will not happen (the MediaPlayer will correctly stay paused when you reactivate it).

**Comment by TheloTheGreat** [comment]
Note: Immediately after you call Pause(), on Windows, IsPlaying=false and IsPaused=true (correct).
But on Mac, immediately after you call Pause(), IsPlaying=true and IsPaused=false (incorrect).

Surely that is the root cause of the issue I described above.

**Comment by TheloTheGreat** [comment]
Fix confirmed on 2.0.10, thanks.
